### PR TITLE
add complete withdrawal button on history without UI

### PR DESCRIPTION
**Pull request by daiki44**
## What?
SSIA
WAK-438

waiting for https://github.com/cryptoeconomicslab/gazelle/pull/359

## TODO
- [x] UI #167 
- [x] function

### CODE REVIEW

**Comment by tkmct** [comment]
it looks like badge (fyi. https://getbootstrap.jp/docs/4.2/components/badge/).
can you make it look actionable to user? more like button.

**Comment by tkmct** [comment]
LGTM

**Comment by daiki44** [comment]
Merge after implementing UI.

File: src/store/withdraw.js [hunk 1]
```diff
@@ -45,52 +50,48 @@ export const withdraw = (amount, tokenContractAddress) => {
...
-const sleep = msec => new Promise(resolve => setTimeout(resolve, msec))
-export const autoCompleteWithdrawal = () => {
-  return async dispatch => {
-    await sleep(20000)
-    dispatch(completeWithdrawal())
-    return await autoCompleteWithdrawal(dispatch)
+      dispatch(getL1Balance())
+      dispatch(getL2Balance())
+      dispatch(getTransactionHistories())
+      dispatch(getPendingExitList())
```
Comment (tkmct) [inline src/store/withdraw.js:85]: What about making these lines in one function?

Comment (daiki44) [reply src/store/withdraw.js:85]: thanks! updated!
5edab20

**Pull request merged**